modules/wizard/wizard.php

```diff
@@ -28,7 +30,7 @@ class PLL_Wizard {
 	/**
 	 * List of steps.
 	 *
-	 * @var array $steps {
+	 * @var array[] $steps {
```

Comment:
Not true according to the doc below:
```suggestion
	 * @var array $steps {
```

Reply:
Fixed by 3edbba947c867926dba47b49dc85b08041b1a019